apps/n8n/1.72.1/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   homeassistant:
-    image: n8nio/n8n:1.72.1
+    image: n8nio/n8n:1.73.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The change from `image: n8nio/n8n:1.72.1` to `image: n8nio/n8n:1.73.1` is a minor version update, which should generally be safe unless you have specific compatibility requirements or dependencies that might break with this upgrade. Consider testing the updated image on a development environment before deploying to production.

If you need any further suggestions for optimizing the file or improving overall configuration, please let me know!